### PR TITLE
Add new system task type SCRIPT_TASK

### PR DESCRIPTION
**Pull request by X-Ultra**
## Description
This new system task type allows execute javascript expression.

## Example use :

```
...
{
  "tasks": [
    {
      "name": "script",
      "taskReferenceName": "script1",
      "type": "SCRIPT_TASK",
      "inputParameters": {
        "input": "${workflow.input}",
        "scriptExpression": "if ($.input.a==1){return {a: true}} else{return {a: false} }"
      }
    }
  ]
}
...
```

### with workflow input:
```
{
  "a": 1
}
```

### output
```
{
    "result":{
        "a":true
    }
}
```

### CODE REVIEW

**Comment by emiteze** [comment]
Looks like this is related to this issue #573 

We are using conductor in our company as orchestration engine and we were considering building something exactly like @X-Ultra did here (many thanks!) to do some simple stuff during workflow execution. What do you guys think of this? @apanicker-nflx @cyzhao @kishorebanala

File: contribs/src/main/java/com/netflix/conductor/contribs/lambda/Lambda.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+        logger.info("scriptExpressionBuilder: {}" , scriptExpressionBuilder);
+
+        try {
+            Object returnValue = null;
+            returnValue = ScriptEvaluator.eval(scriptExpressionBuilder, taskInput);
+            taskOutput.put("result", returnValue);
+            task.setStatus(Task.Status.COMPLETED);
+        } catch (ScriptException e) {
+            logger.error(e.getMessage(), e);
```
Comment (apanicker-nflx) [inline contribs/src/main/java/com/netflix/conductor/contribs/lambda/Lambda.java]: Log a better error message here, the message inside the exception is already printed out in the stack trace.

File: contribs/src/main/java/com/netflix/conductor/contribs/lambda/Lambda.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+import com.netflix.conductor.core.execution.tasks.WorkflowSystemTask;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Singleton;
+import javax.script.ScriptException;
+import java.util.Map;
+
+@Singleton
+public class Lambda extends WorkflowSystemTask {
```
Comment (kishorebanala) [inline contribs/src/main/java/com/netflix/conductor/contribs/lambda/Lambda.java]: @X-Ultra One more nitpick - Can you please rename class to `LambdaTask`, while still keeping the task type as `Lambda`. You can check `HttpTask` for reference. Also, please add javadoc for this class, briefly explaining how this task is supposed to be used. We'll migrate this over to documentation.

Comment (X-Ultra) [reply contribs/src/main/java/com/netflix/conductor/contribs/lambda/Lambda.java]: Okay, it doesn't matter. ~

**Comment by kishorebanala** [comment]
@X-Ultra Looks like you've merged dev branch into your branch, and there are lots of commits unrelated to this PR. Can you please rebase your branch and clean this up. Having commits and changes that are already merged makes it really difficult to review. Thank you.

**Comment by X-Ultra** [comment]
> @X-Ultra Looks like you've merged dev branch into your branch, and there are lots of commits unrelated to this PR. Can you please rebase your branch and clean this up. Having commits and changes that are already merged makes it really difficult to review. Thank you.

ok, later.

**Pull request closed without merging**